.prettierignore

```diff
@@ -0,0 +1 @@
+*.mp3
```

Comment:
Add a newline at the end to make the red symbol go away.

Also configure your editor to add a newline to files automatically